ds_judgements_public_ui/templates/pages/terms_of_use.html

```diff
@@ -14,76 +14,61 @@
-    </div>
-    <div>
-      Accusamus, aperiam aut autem error explicabo incidunt ipsam neque obcaecati omnis, quaerat qui sed sit
-      totam ut voluptate! Amet distinctio dolor ducimus inventore minima nihil odio quaerat quam recusandae totam.
-    </div>
-    <h2>Lorem ipsum dolor sit amet.</h2>
-    <div>
-      Accusamus at dolore dolorem enim eos error et explicabo fugit harum id ipsa labore natus nostrum, obcaecati
-      quia quis veniam voluptas voluptate, voluptatem, voluptatum. Error excepturi fuga nisi quam vitae!
-    </div>
-    <div>
-      Dolores ea eaque eius enim ipsum maiores nostrum optio quia rerum suscipit. Error ipsum iste magnam, quis
-      quisquam repellat vitae? Aliquid corporis debitis dolor, earum esse pariatur perspiciatis recusandae sint.
-    </div>
-    <div>
-      Adipisci consequatur enim quae quisquam! Accusamus ad adipisci amet consequatur dicta dignissimos dolorem
-      eaque illum itaque iure magnam magni nam nemo, neque qui quibusdam quisquam, quod sed, tenetur vitae.
-      Dolore!
-    </div>
-    <div>
-      Adipisci, aut beatae dicta earum, eius est et incidunt iste labore libero maxime minima modi mollitia nam
-      pariatur perspiciatis quam repellendus reprehenderit sed voluptate! Accusantium dolores eius optio quae sit!
-    </div>
-    <div>
-      Aliquid asperiores consequatur dicta dignissimos esse expedita fuga in, iusto optio, possimus quidem, quos
-      recusandae repellat. Asperiores consequatur delectus in iusto? Aspernatur, est explicabo fuga minima
-      perspiciatis possimus rem saepe.
-    </div>
-    <h2>Lorem ipsum dolor sit amet.</h2>
-    <div>
-      A, alias aliquam at, culpa dignissimos distinctio dolor ducimus ea fugit impedit in ipsa iusto labore
-      laborum natus nemo nesciunt perferendis porro quae quaerat quam quasi rem saepe tempora vel.
-    </div>
-    <div>
-      Dolorum nesciunt, praesentium! Aperiam, aspernatur enim eveniet incidunt minima possimus quos? Adipisci at
-      deserunt dolorem doloremque eligendi fugiat illum, impedit inventore, ipsum nostrum, numquam odit quam
-      tempore vel vero voluptas.
-    </div>
-    <div>
-      Ab amet aperiam cum, dolorum ea eius eligendi error esse illo itaque iure laboriosam, maiores maxime
-      necessitatibus non optio pariatur possimus quis quisquam quo totam ut voluptas voluptates voluptatibus
-      voluptatum!
-    </div>
-    <div>
-      Asperiores, autem commodi consequatur dicta ea error est fuga mollitia nisi, placeat ratione sequi soluta,
-      suscipit. Assumenda consectetur dolores nemo nobis pariatur quae recusandae voluptatum? Laborum mollitia
-      quaerat quam ullam?
-    </div>
-    <div>
-      Consequuntur dignissimos ea excepturi necessitatibus non officia similique ut! Alias numquam officia
-      provident tempore vero! A amet consequatur dolorum magni nam. Ab earum eum fuga numquam quod repellat
-      voluptas! Rerum.
-    </div>
+
+    <h2>What is in these terms</h2>
```

Comment:
I think this text should be put through the translation service, so that it's added to the `en_GB/django.po` file